misc_datasets/verify/betweennesscentrality.ldbc003.sourceFile

```diff
@@ -0,0 +1,100 @@
+4113 
```

Comment:
These sources are a parameter to the app. 

For queries, we store the parameters in the main repo:
https://github.com/KatanaGraph/katana-enterprise/tree/master/data
I would suggest you add a new folder for analytics there but I'm not opposed to storing it in test-datasets.